src/components/RecommendedAlbum/RecommendedAlbum.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './RecommendedAlbum.css'
+
+const RecommendedAlbum = ({ id, title, coverImg }) => {
+
+    return (
+        <Link to={`album/${id}`} className='link'>
```

Comment:
this needs a forward slash and the current route is "albums" 

 <Route path='/albums/:id' element={<Album/>}/>


 so `/albums/${id}`

Could change route to "album", link to on card would need to be updated.